packages/core/admin/admin/src/pages/SettingsPage/pages/License/PageView/index.js

```diff
@@ -0,0 +1,257 @@
+        true
+      );
+
+      if (status === 'ok') {
+        setShouldFetch(true);
+        setSavedLicense(license);
+        toggleNotification({
+          type: 'success',
+          message: { id: messageId },
+        });
+      } else {
+        toggleNotification({
+          type: 'warning',
+          message: { id: messageId },
+        });
+      }
+    } catch (error) {
+      console.log(error);
+    } finally {
+      setSavedLicense('');
+      setLicense('');
+      unlockAppWithAutoreload();
+    }
+  };
+
+  console.log(globalData.licenseInfo.expireAt);
+
+  return (
+    <Layout>
+      <SettingsPageTitle name="Webhooks" />
+      <Main aria-busy={isLoading}>
+        <HeaderLayout
+          title={formatMessage({
+            id: 'Settings.license.title',
+            defaultMessage: 'License Activation',
+          })}
+          subtitle={formatMessage({
+            id: 'Settings.license.description',
+            defaultMessage: 'Quickly activate your EE license',
+          })}
+          primaryAction={
+            <>
+              {globalData.disabled && canRead && canCreate ? (
+                <></>
+              ) : (
+                <Button
+                  startIcon={<Check />}
+                  disabled={globalData.disabled || license.length <= 0 || license === savedLicense}
+                  variant="default"
+                  onClick={saveLicense}
+                  size="L"
+                >
+                  {formatMessage({
+                    id: 'Settings.license.button.save',
+                    defaultMessage: 'Save',
+                  })}
+                </Button>
+              )}
+            </>
+          }
+        />
+        <ContentLayout>
+          {isLoading ? (
+            <Box background="neutral0" padding={6} shadow="filterShadow" hasRadius>
+              <LoadingIndicatorPage />
+            </Box>
+          ) : (
+            <>
+              <Stack spacing={7}>
+                <Box background="neutral0" hasRadius shadow="filterShadow">
+                  <ContentBox
+                    title={formatMessage({
+                      id: 'Settings.license.information',
+                      defaultMessage: 'Information',
+                    })}
+                    subtitle={`${formatMessage({
+                      id: globalData.messageId,
+                      defaultMessage: '',
+                    })}. ${
+                      globalData.licenseInfo
+                        ? `${formatMessage({
+                            id: 'Settings.license.information.license.edition',
+                            defaultMessage: 'Edition',
+                          })} ${globalData.licenseInfo.type} - ${formatMessage({
+                            id: 'Settings.license.information.license.expires_on',
+                            defaultMessage: 'Expires on',
+                          })}: ${moment(globalData.licenseInfo.expireAt).format('MM/DD/YYYY')}`
```

Comment:
Throws an error if no license is installed

Reply:
Yeah, no env var and no license.txt

Could also try an invalid license.

Reply:
Could you use date-fns instead of moment